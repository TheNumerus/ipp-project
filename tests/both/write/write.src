.ippcode20
WRITE string@constants:\010

WRITE int@00000012345
WRITE string@\010

WRITE int@-0000012345
WRITE string@\010

WRITE bool@true
WRITE string@\010

WRITE bool@false
WRITE string@\010

WRITE nil@nil
WRITE string@\010

WRITE string@variables:\010
DEFVAR GF@write

MOVE GF@write int@00000012345
WRITE GF@write
WRITE string@\010

MOVE GF@write int@-0000012345
WRITE GF@write
WRITE string@\010

MOVE GF@write bool@true
WRITE GF@write
WRITE string@\010

MOVE GF@write bool@false
WRITE GF@write
WRITE string@\010

MOVE GF@write nil@nil
WRITE GF@write
WRITE string@\010