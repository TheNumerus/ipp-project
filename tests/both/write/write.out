constants:
12345
-12345
true
false

variables:
12345
-12345
true
false

